docs/sources/reference/builder.md

```diff
@@ -349,9 +349,9 @@ accessible from the host by default. To expose ports to the host, at runtime,
     ENV <key>=<value> ...
 
 The `ENV` instruction sets the environment variable `<key>` to the value
-`<value>`. This value will be passed to all future 
-`RUN`, `ENTRYPOINT`, and `CMD` instructions. This is
-functionally equivalent to prefixing the command with `<key>=<value>`
+`<value>`. This value will be in the environment of all "descendent" builds
```

Comment:
s/builds/Dockerfile commands/
The first time I read "descendent builds" I interpreted it as "the next time I run 'docker build' ".  Could just be me though.